src/network/networkd-dhcp-server.c

```diff
@@ -1,8 +1,9 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
 #include <netinet/in.h>
-#include <linux/if_arp.h>
-#include <linux/if.h>
+#include <net/if.h>
+#include <linux/if_arp.h> /* Must be included after <net/if.h> */
+#include <linux/if.h> /* Must be included after <net/if.h> */
```

Comment:
hmm, we used to have trouble with things like this? has this come back? the includes so far were fine tuned to not trigger these conflicts between glibc and kernel headers. what changed?

Reply:
The kernel headers are tuned, the glibc headers are not. I've ran into this before, I don't know why the glibc peeps don't fix their headers.